commands/web/phpstan

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#ddev-generated
+## Command provided by https://github.com/ddev/ddev-drupal-contrib
+## Description: Run phpstan inside the web container
+## Usage: phpstan [flags] [args]
+## Example: "ddev phpstan" or "ddev phpstan -n"
+## ProjectTypes: drupal,drupal8,drupal9,drupal10
+## ExecRaw: true
+
+if ! command -v phpstan >/dev/null; then
+  echo "phpstan is not available. You may need to 'ddev poser'"
+  exit 1
+fi
+test -e phpstan.neon || curl -OL https://git.drupalcode.org/project/gitlab_templates/-/raw/default-ref/assets/phpstan.neon
+# Add an empty baseline file to ensure it exists.
+test -e phpstan-baseline.neon || touch phpstan-baseline.neon
+export COMPOSER=composer.contrib.json
+.ddev/commands/web/expand-composer-json "$DDEV_PROJECT_NAME"
```

Comment:
This command should assume that composer has already been run. Thats how phpunit command works, for example. So we can remove these lines.